Gemfile

```diff
@@ -1,7 +1,7 @@
 # encoding: utf-8
 source "https://rubygems.org"
 
-if RUBY_VERSION =~ /^2\.2\.[0-1]p?\d*/ || RUBY_VERSION =~ /^2\.1\.\d*p?\d*/
+if RUBY_VERSION =~ /^2.[3-5].\d*p?\d*/ || RUBY_VERSION =~ /^2\.2\.[0-1]p?\d*/ || RUBY_VERSION =~ /^2\.1\.\d*p?\d*/
```

Comment:
Do you need this?  The tests seem to pass without it.  These blocks are more of exceptional cases.  Ideally we just pull from the gemspec.